packages/kbn-ui-framework/package.json

```diff
@@ -31,7 +31,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.11.1",
-    "@elastic/eui": "0.0.55",
```

Comment:
🤦‍♂️

Reply:
Do the ui-framework docs still work with this change?

Reply:
That's a good point - We have a discussion to remove ui-framework entirely in https://github.com/elastic/kibana/issues/46410#issuecomment-697558570.

I will revert this line, and create another PR to remove all but the assets we're currently relying on for ui-framework.